nginx/changelog.d/16720.added

```diff
@@ -0,0 +1 @@
+Nginx: suppport the upstream response time if use_vts is true
```

Comment:
```suggestion
Suppport the upstream response time if use_vts is true
```

Nginx will have its own changelog file so this should be enough

Reply:
@FlorentClarret Thanks, I fixed it.